### PR TITLE
Fix getting MAC address on Windows

### PR DESCRIPTION
**Pull request by malikm**
Also resolves JAVA 7 dependency (#497).

Signed-off-by: Martin Malik <martin.malik@eurotech.com>

### CODE REVIEW

File: kura/org.eclipse.kura.core/src/main/java/org/eclipse/kura/core/system/SystemServiceImpl.java [hunk 1]
```diff
@@ -573,8 +573,9 @@ public String getPrimaryMacAddress() {
 				Enumeration<NetworkInterface> networks = NetworkInterface.getNetworkInterfaces();
 				while (networks.hasMoreElements()) {
 					NetworkInterface network = networks.nextElement();
-					if(network.getIndex() == 0) {
+					if(network.getName().equals("eth0")) {
 						ip = network.getInetAddresses().nextElement();
```
Comment (cdealti) [inline kura/org.eclipse.kura.core/src/main/java/org/eclipse/kura/core/system/SystemServiceImpl.java]: @malikm As a matter of style we encourage using:
``"eth0".equals()`` which will not throw NPE if ``network.getName()`` happens to return ``null``. Can you please rework it?

Comment (malikm) [inline kura/org.eclipse.kura.core/src/main/java/org/eclipse/kura/core/system/SystemServiceImpl.java]: Done

**Comment by cdealti** [comment]
@malikm can you please squash f276a20 and 79cdc7c and remove 9b38cad? Use ``git rebase -i`` for example  ``git rebase -i HEAD~3`` (the three most recent commits - you may need to change this number). When you are done force a push on your PR branch (``git push -f origin pr/fix497``)

**Comment by malikm** [comment]
Done.

**Pull request merged**